### PR TITLE
Version 4

### PR DESCRIPTION
**Pull request by rowanmanning**
Each of the commits in here have been reviewed separately, and a [migration guide is available here](https://github.com/Financial-Times/o-labels/tree/4.x#migration-guide).

## Master

screenshot 2019-02-07 at 13 13 56

## Internal

screenshot 2019-02-07 at 13 14 37

### CODE REVIEW

**Comment by JakeChampion** [comment]
Is this waiting on approval from someone other than @Financial-Times/origami-core ?

**Comment by apaleslimghost** [comment]
the App is currently using the `active` and `error` states, although this isn't consistent with ft.com's current labelling of live blogs, so this is probably a good point for it to not be using those labels. at one point the `live` (and possibly `pending`) state was being used _[somewhere on ft.com trying to find out]_ but i don't think they are any more.

**Comment by rowanmanning** [comment]
This is still needing some small changes.
- [x] The screenshot needs updating
- [x] The main `oLabels` mixin needs to have a single map option
- [x] Make changes to the state mixin based on discussion with the team
- [x] Move to a separate migration guide

File: src/scss/_mixins.scss
Comment (gvonkoss) [inline src/scss/_mixins.scss]: @notlee and I have chatted about file structure and such before  — we've agreed to put the primary mixin in the `main.scss` file, in order to make it easier to find and consistent across our components 😬

**Comment by notlee** [comment]
New labels with controls on Heimdall https://heimdall.in.ft.com/dashboard?teamid=origami-team

screenshot 2018-12-14 at 13 53 57

**Comment by notlee** [comment]
More labels! Lodestar.

screenshot 2019-01-25 at 11 37 29

**Review by notlee — APPROVED** [review]

**Pull request merged**
